applications/matter_bridge/boards/nrf54h20dk_nrf54h20_cpuapp.conf

```diff
@@ -9,10 +9,6 @@ CONFIG_CHIP_QSPI_NOR=n
 
 CONFIG_MPU_STACK_GUARD=n
 
-# Enable PSA crypto from SSF client
-CONFIG_PSA_SSF_CRYPTO_CLIENT=y
```

Comment:
Why have those lines been removed from this sample only? I can see that you've added them to the other WiFi samples, is it ok?

Reply:
Contacting you offline about this...